test/mpi.jl

```diff
@@ -28,15 +29,21 @@ params = Dict(
 )
 
 # Warmup
-rm("particle_da.h5"; force = true)
 run_particle_filter(Model.init, params, BootstrapFilter())
 # Flush a newline
 println()
 
 # Run the command
-rm("particle_da.h5"; force = true)
+params["filter"]["output_filename"] = "bootstrap_filter.h5"
```

Comment:
Maybe drop a comment here to remind us why we want to enable writing the output file?  Something like
```suggestion
# Write output file to make sure it works correctly when using multiple MPI ranks
params["filter"]["output_filename"] = "bootstrap_filter.h5"
```